### ISSUE TITLE
subsampling for gblinear booster

### ISSUE THREAD

**Issue by zl365** [dmlc/xgboost#5379]
In the documentation, subsample is a parameter appears in the list of parameters for tree boosters only. Please also include subsampling for gblinear booster as well. Subsampling of training instances can be very useful for linear boosters, especially there are a large number of predictors.  It probably only need a small modification after the gradients are calculated.  

I have tried to set the subsample parameter at different values with a gblinear booster, and I got same results (estimated parameter values and predicted values). So perhaps, subsampling only work with tree boosters in current version of xgboost. If I am mistaken, could anyone please tell me how to implement subsampling o training instances with gblinear booters correctly, thank you very much.

**Comment by trivialfis** [comment]
Subsampling is currently not supported by linear booster.

**Comment by trivialfis** [comment]
Closing as GBLinear has been deprecated https://github.com/dmlc/xgboost/issues/12023.
